RELEASE/scripts/autoscend/auto_community_service.ash

```diff
@@ -143,7 +143,7 @@ boolean LA_cs_communityService()
 			print("A Wanderer event is expected now, we want to re-digitize", "blue");
 			auto_sourceTerminalEducate($skill[Digitize], $skill[Extract]);
 			set_property("auto_combatDirective", "start;skill digitize");
-			if(!cs_mpMaintain() || !cs_healthMaintain()){
```

Comment:
I'm curious about the reasoning on swapping the order on these - would be somewhat worried about "restore up to 40 MP" followed by "cocoon to restore HP" leaving us with an unexpectedly low amount of HP for combat. But that's low-confidence, was just wondering your thoughts on this.

Reply:
acquireHP will try to restore enough mp to cast a skill if it needs to. if we do mp first we should end up in the exact same place but potentially go into battle with less mp than we want since we used it healing.